modules/lightmapper_rd/lm_compute.glsl

```diff
@@ -264,7 +264,17 @@ uint trace_ray(vec3 p_from, vec3 p_to, bool p_any_hit, out float r_distance, out
 			break;
 		}
 
-		bvec3 mask = lessThanEqual(side.xyz, min(side.yzx, side.zxy));
+		// There should be only one axis updated at a time for DDA to work properly.
+		// bvec3 mask = lessThanEqual(side.xyz, min(side.yzx, side.zxy));
```

Comment:
```suggestion
		//bvec3 mask = lessThanEqual(side.xyz, min(side.yzx, side.zxy));
```
Commented code should be without space to differentiate

Reply:
Thanks!